<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectDesignTypeForm</class>
 <widget class="QWidget" name="SelectDesignTypeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="m_labelTitle">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Create New Runs</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_labelDetail">
     <property name="text">
      <string>This wizard will guide you through the process of creating and launching multiple synthesis or/and implementation runs. You can then apply diffent sets of command options,or strategies, to these runs and launch them all at once.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>25</number>
     </property>
     <property name="leftMargin">
      <number>15</number>
     </property>
     <property name="topMargin">
      <number>15</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="m_labelOption">
       <property name="text">
        <string>What type runs do you want to create?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_radioBtnSynth">
       <property name="text">
        <string>  Synthesis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_radioBtnImple">
       <property name="text">
        <string>  Implementation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_radioBtnBoth">
       <property name="text">
        <string>  Both</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>584</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
